Makefile

```diff
@@ -65,16 +60,18 @@ fallthrough: submodules
 	@echo Initial setup complete. Running make again . . .
 	@make
 
-
 # Generate a coverage report for cobertura applying exclusions on
 # - generated file
 cobertura:
 	@cat $(GO_TEST_OUTPUT)/coverage.txt | \
 		grep -v zz_generated.deepcopy | \
 		$(GOCOVER_COBERTURA) > $(GO_TEST_OUTPUT)/cobertura-coverage.xml
 
+crds.clean:
+	@sed -i '1,2d' package/crds/*.yaml
```

Comment:
This removes the first two lines of each CRD, which is output as:
```

---
```
This is because `pkg-unpack` will read this as two different YAML documents, and one of them will not be a valid k8s object. We could build logic into it to say skip if it is a blank YAML document, but I actually don't think it is horrible to have it work as is (i.e. a blank YAML document is not a valid thing to include in a package), except for the fact that `controller-gen` outputs in this format :roll_eyes: